### ISSUE TITLE
Cleanup (uninstall) of installed files

### ISSUE THREAD

**Issue by pranav-g10** [jontewks/puppeteer-heroku-buildpack#35]
@jontewks I'm using puppeteer for the purpose of crawling and storing static version of my React single-page-app in the disk for the benefits of bots/crawlers. The crawling process uses [react-snap](https://www.npmjs.com/package/react-snap) and I've integrated it into my build process using another buildpack: [react-snap-heroku-buildpack](https://github.com/pranav-g10/react-snap-heroku-buildpack).

tl;dr version of above: I need puppeteer solely during the heroku build process and never after that.

So can you provide a list of commands that ought to be run to uninstall all the libs/binaries installed by your buildpack ? I'll incorporate those commands in my own buildpack, essentially uninstalling puppeteer once the local crawling succeeds and thus reduce my slug size.
